greenheart/tools/eco/hopp_mgmt.py

```diff
@@ -148,6 +148,22 @@ def setup_hopp(
                     hopp_technologies["wind"][key]
                 )
 
+
+    # setup hopp interface
+    hopp_config_internal = copy.deepcopy(hopp_config)
```

Comment:
Is this deepcopy necessary?

Reply:
I did this to preserve the cost values for wave since I pop those off to later set up wave. I think we should fix this conflict of methods in HOPP before removing the deep copy